### PR TITLE
feat(simon, simone): Add `collarWidth` for simon(e)

### LINKED ISSUE #6907: [feature]: Add option for collar width multiplier in Simon
### What is it that you would like to see happen? 🤔

In Simon, currently a fixed 1.2 multiplier is used to calculate the collar width from the collar stand width. For smaller sizes, this results in a collar that barely covers the stand without a tie and does not cover the collar stand with a thicker tie. An option to adjust and increase this multiplier would allow users to resolve this issue in their patterns.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by VSinerva**
Closes #6907. Allows the user to adjust the multiplier used to derive the collar width from the collar stand width. This can help the collar cover the collar stand. The design seemed to adapt fine, so I made the range fairly broad to enable unforeseen uses of the option. First time I'm adding *anything* to a design, so please let me know all the places I messed up :rofl:

### CODE REVIEW

File: designs/simon/src/collar.mjs [hunk 1]
```diff
@@ -24,7 +25,7 @@ function simonCollar({
 }) {
   const draft = function (tweak = 1) {
     const length = measurements.neck * (1 + options.collarEase - options.collarGap) * tweak
-    const width = store.get('collarStandWidth') * 1.2 * (1 + options.collarRoll)
+    const width = store.get('collarStandWidth') * options.collarWidth * (1 + options.collarRoll)
```
Comment (HaasJona) [inline designs/simon/src/collar.mjs:28]: Looks like the "collarRoll" setting already does basically the same as the new option. It just has a super small range and applies on top of the 1.2 factor. Not sure if anything needs to be done with that redundancy.

**Review by HaasJona — COMMENTED** [review]
Looks good to me. Added a few comments that I'm not sure about.

File: markdown/org/docs/designs/simon/options/collarwidth/de.md
[the remark this answers is not in the document]
Comment (VSinerva) [reply markdown/org/docs/designs/simon/options/collarwidth/de.md:1]: I was under the impression that this was only the case when changing existing files. Not when adding new files. I think you have to "init" the new files by copying the `en.md`, and then you can let crowdin manage any later diffs.

[the remark this answers is not in the document]
Comment (HaasJona) [reply markdown/org/docs/designs/simon/options/collarwidth/de.md:1]: For my completely new designs (e.g. Bibi) I was told to only create the en files and that worked fine, the others were added by crowdin. I'm not sure if they hurt, though.

File: designs/simon/src/collar.mjs (hunk 1 above)
Comment (VSinerva) [reply designs/simon/src/collar.mjs:28]: Good catch! I'd mistaken `collarRoll` for something else. Based on the description, I thought it was used for the line for the undecollar, to make the collar "roll" back and hide the seam between the two near-identical collar pieces. I don't think having both options is necessary. Maybe this PR should be replace with clarifying + expanding the doc for `collarRoll`, and expanding it's range?

Though getting rid of the fixed 1.2 multiplier in the code and changing the default for `collarRoll` to compensate would likely be a breaking change that those who used a custom `collarRoll` would have to compensate for w/old patterns. I'm not sure what the best way to handle such a situation is.

File: designs/simon/src/options.mjs [hunk 2]
```diff
@@ -35,6 +35,7 @@ export const collarBend = { pct: 3.5, min: 0, max: 10, menu: 'style.collar' }
 export const collarFlare = { deg: 3.5, min: 0, max: 10, menu: 'style.collar' }
 export const collarGap = { pct: 2.5, min: 0, max: 6, menu: 'style.collar' }
 export const collarRoll = { pct: 3, min: 0, max: 6, menu: 'style.collar' }
+export const collarWidth = { pct: 120, min: 10, max: 300, menu: 'style.collar' }
```
[the remark this answers is not in the document]
Comment (VSinerva) [reply designs/simon/src/options.mjs:38]: Seeing as the timeline for out-of-range values is "TBD" and "contingent on several major changes", I felt it best not to make decisions based on it, and thus didn't want to artificially limit users too much, since the pattern adapts gracefully either way. I'm still down for narrowing the range, if desired.

File: markdown/org/docs/designs/simon/options/collarwidth/de.md
[the remark this answers is not in the document]
Comment (VSinerva) [reply markdown/org/docs/designs/simon/options/collarwidth/de.md:1]: Good to know! I doubt they hurt anything, since I've previously had PRs merged that made changes to the other language files (before I knew better). Still, saves some copying on my end :)

[the remark this answers is not in the document]
Comment (VSinerva) [reply markdown/org/docs/designs/simon/options/collarwidth/de.md:1]: I removed the extra files, just to clean up the PR a bit. Thanks for pointing this out! :)

File: designs/simon/src/options.mjs (hunk 2 above)
[the remark this answers is not in the document]
Comment (VSinerva) [reply designs/simon/src/options.mjs]: After sleeping on this, I agree with the smaller range being more sensible. Ultimately, the limits are also part of "our" communication with the user, and having such a broad range makes that communication less clear.

File: designs/simon/src/collar.mjs (hunk 1 above)
Comment (VSinerva) [reply designs/simon/src/collar.mjs:28]: I made a commit that removes the redundancy, by having `collarRoll` affect the undercollar line. It seems to me this was the intended use, *or* there's an undercollar <-> collar stand mixup *somewhere* that is making `collarRoll` a very confusing option to me. Once joost reviews the PR, I'll let him decide if this is indeed a fix, or if there's another way this situation should be handled :D

Comment (karlnippoldt) [reply designs/simon/src/collar.mjs:28]: I don’t think the collar roll option pertains to the issue I filed #6907 about. Collar roll seems to just affect the difference between the collar and undercollar. I would like to be able to adjust the height of the collar/undercollar and collar stand separately, instead of them always changing together like they are now.

**Review by joostdecock — APPROVED** [review]

**Comment by joostdecock** [comment]
Thanks for this @VSinerva as well as everyone weighing in on this.

The `collarRoll` option is indeed supposed to control the difference between upper and under collar.

The height itself was previously determined by the collar stand height alone, (using a fixed value to make the collar higher/wider, so this PR is a clear improvement.

As for breaking changes, in general we do not consider changes that change a design as breaking, as that would mean we'd probably have to make most changes a major release even though there's no breaking changes from a code point of view. So it would make the semantic versioning everything but.

TL;DR: this is great improvement and I'm very appreciative of this contribution.
Forgive me for taking so long to get around to this, I'm dealing with a lot right now 🙏
